tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.yaml

```diff
@@ -86,6 +86,7 @@ Resources:
     Condition: CreateFsx
     Properties:
       FileSystemType: LUSTRE
+      FileSystemTypeVersion: '2.12'
```

Comment:
Why is this change required just now?

Reply:
Lustre compatibility: https://docs.aws.amazon.com/fsx/latest/LustreGuide/lustre-client-matrix.html
Basically, RHEL9 uses Lustre client 2.15, which supports Lustre server 2.12+